private-samples/excel/01-basics/basic-modified-api-call.yaml

```diff
@@ -1,4 +1,4 @@
-id: excel-basic-api-call
```

Comment:
Do we actually need this one at all?  Or at least, can you name the file with a "TEMP" prefix, to make it clear that you're only doing this for testing, and that the file can be deleted as soon as we have something real to put in that folder instead?